kits/opportunity-legitimacy-screener/prompts/score-and-explain_llmnode-353_user_1.md

```
@@ -0,0 +1,7 @@
+Company: {{codeNode_709.output.signals.company_name}}
```

markdownlint-cli2 (0.23.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)